server/controllers/rpaFrameworkCommandsController.js

```diff
@@ -1,8 +1,8 @@
-const activityDataRetrieval = require('../services/activityDataRetrieval');
+const activityDataRetrieval = require('../services/ActivityDataRetrieval');
 
 exports.getAvailableApplications = async (req, res) => {
   try {
-    let listOfDistinctApplications = await activityDataRetrieval.getDistinctApplicationsFromDB();
+    const listOfDistinctApplications = await activityDataRetrieval.getDistinctApplicationsFromDB();
     res.set('Content-Type', 'application/json');
     res.send(listOfDistinctApplications);
   } catch (err) {
```

Comment:
I cant select the next line because of the linting warning, but wanted to ask, if it is possible to ignore those console.log statements where clearly an error is being logged in an error case. Not sure, if that is possible, just a quick thought I had

Reply:
There is a possibility, but I think filtering of the console.log is necessary, to find unessecary logs. I could adapt the rule so that [console.warn and console.error will be waved through](https://eslint.org/docs/rules/no-console). What do you think about that?